### DISCUSSION TITLE
Change response_type on OIDC

### DISCUSSION THREAD

**Discussion by HappyMoai** [keycloak/keycloak#22479] [Q&A] [answered]
Hello,
How do I change the response_type querry parameter of my OIDC?
I have added an OIDC Identity Provider, but the default response_type is "code". How do I change this?

**Comment by justin-tay** [comment, marked as answer]
If you are brokering to an OpenID Connect Provider the `response_type` can only be `code` and cannot be changed by configuration.

https://github.com/keycloak/keycloak/blob/48a4102c9700ed50034f2560078b7b9d0cd1e952/services/src/main/java/org/keycloak/broker/oidc/AbstractOAuth2IdentityProvider.java#L320-L326
